src/handler.py

```diff
@@ -130,10 +137,11 @@ def main():
     busy_airports = ['MAD', 'CDG', 'AMS', 'FCO', 'DUB']
     n_tries = 2
     day_count = 180
-    input = [(airport, n) for airport in busy_airports for n in range(day_count)]
-    np.random.shuffle(input)
-    for arrival, n in input:
+    inputs = [(airport, n) for airport in busy_airports for n in range(day_count)]
+    np.random.shuffle(inputs)  # TODO: what's the shuffle for? In case choosing one first affect the price?
```

Comment:
Because it takes 12 hours to run- so if we shuffle we'll get a good spread of requests over different times of day